### ISSUE TITLE
Waves plugins not detected after upgrading to v10

### ISSUE THREAD

**Issue by lentferj** [osxmidi/LinVst#41]
After upgrading Waves to v10 you end up with WaveShell1-VST 10.0_x64.dll. After putting the respective "WaveShell1-VST 10.0_x64.so" in the same folder scanning fails on Mixbus4. I tried with updating via Waves Central, and also via removing all installed plugins, moving license back to the Waves cloud and using "Easy install & activate" from Waves Central.

**Comment by osxmidi** [comment]
This might be due to a Wine dll issue if the new Waves version is using different dll's to V9, it could also be due to if Waves installed their addition dll's ok.

I will need to test it.

**Comment by osxmidi** [comment]
I tested Waves v10 and got the same result as you did.

So I tested some things and eventually got V10 to work.

linvst needs to go into  ~/.wine/drive_c/Program Files/VSTPlugIns and then moved to the waveshell name mv linvst.so "WaveShell1-VST 10.0_x64.so"

Then set the DAW to scan  ~/.wine/drive_c/Program Files/VSTPlugIns and start the DAW and a select folder dialog will appear.

Descend into the path by double clicking on ( C:)
and then double clicking on Program Files (x86)
and then double clicking on Waves
and then just highlight Plug-Ins V10 with a single click (don't descend into it)
and finish.

Then the DAW scan should pick up the Waves plugins.
